src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.scss

```diff
@@ -0,0 +1,25 @@
+@import 'variables';
+
+:host {
+    background-color: white;
+    width: 100%;
+
+    ::ng-deep {
```

Comment:
Are you sure we need this code in this ng-deep? Did you see this code if you run and inspect those elements?

Reply:
yes it's needed otherwise the sidebar's header won't extend